extensions/amp-animation/0.1/install-polyfill.js

```diff
@@ -13,32 +13,43 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import {Deferred} from '../../../src/utils/promise';
 import {Services} from '../../../src/services';
-import {installWebAnimations} from 'web-animations-js/web-animations.install';
 
-const POLYFILLED = '__AMP_WA';
+const polyfillPromiseMap = new Map();
```

Comment:
Should we do `WeakMap`? Otherwise FIE are destroyed regularly.

Reply:
Opted not to because of low support. I'll follow up in another PR after a bit of analysis.